.github/workflows/vllm_ascend_test.yaml

```
@@ -202,6 +202,7 @@ jobs:
             VLLM_USE_MODELSCOPE=True pytest -sv tests/multicard/test_offline_inference_distributed.py::test_models_distributed_topk
             VLLM_USE_MODELSCOPE=True pytest -sv tests/multicard/test_offline_inference_distributed.py::test_models_distributed_DeepSeek_W8A8
             VLLM_USE_MODELSCOPE=True pytest -sv tests/multicard/ --ignore=tests/multicard/test_ilama_lora_tp2.py --ignore=tests/multicard/test_offline_inference_distributed.py
+	    VLLM_USE_MODELSCOPE=True pytest -sv tests/multicard/test_model_DeepSeek_W4A8.py::test_model_w4a8
```

```suggestion
            VLLM_USE_MODELSCOPE=True pytest -sv tests/multicard/test_model_DeepSeek_W4A8.py::test_model_w4a8
```

This break CI: https://github.com/vllm-project/vllm-ascend/actions/runs/15755992760/workflow